rosbag2_py/test/test_transport.py

```diff
@@ -84,6 +84,6 @@ def test_record_cancel(tmp_path, storage_id):
 
     metadata = metadata_io.read_metadata(bag_path)
     assert len(metadata.relative_file_paths)
-    storage_path = Path(metadata.relative_file_paths[0])
+    storage_path = Path(bag_path + '/' + metadata.relative_file_paths[0])
```

Comment:
I think it might be better to refactor `bag_path` a bit, so we can use the overloaded `/` operator more effectively.

In particular, on line 47, we should probably change this to:
```
bag_path = tmp_path / 'test_record_cancel'
```

Then in lines 48 and 81, change it to use `str(bag_path)` as necessary.

Finally, in this line (line 87), we should be able to do:

```
storage_path = bag_path / metadata.relative_file_paths[0]
```

Reply:
@clalancette Thanks for the additional review. I've addressed your suggestion in the https://github.com/ros2/rosbag2/pull/1592/commits/c47b619bb389120023dca069ad4630ea2b3e1ce8